core/metamodel/src/main/java/org/apache/isis/core/metamodel/adapter/ObjectAdapterProvider.java

```diff
@@ -35,6 +37,12 @@
     
     // -- INTERFACE
 
+//    /**
+//     * @param pojo
+//     * @return oid for the given domain object 
+//     */
+//    Oid oidFor(Object domainObject);
+    
```

Comment:
delete?

Reply:
OK, I understand ... this is code on the way to being introduced, rather than on the way to being removed.

Reply:
exactly!